apps/api/src/domains/feedback/feedback.mysql.service.ts

```diff
@@ -287,6 +288,7 @@ export class FeedbackMySQLService {
 
           return query;
         },
+        updatedAt: () => `'${dayjs().format('YYYY-MM-DD HH:mm:ss')}'`,
```

Comment:
@chiol 
Using `createQueryBuilder`, there is no choice but put `updatedAt` directly to the query builder.